docs/migrate/migrate-estimator.rst

```diff
@@ -0,0 +1,201 @@
+
+Sometimes the system is small enough that we can compute the expectation value classically. With opflow, this was done by composing the circuit and operator states, then calling for the exact evaluation method:
+
+
+.. code-block:: python
+
+    opflow_state_func = opflow_state.adjoint().compose(opflow_op).compose(opflow_state)
+    expectation_value_1 = opflow_state_func.eval().real # easy expectation value, use for small systems only!
+
+    print("exact: ", expectation_value_1)
+
+Option 2: Construct the expectation circuit and sample on a system or simulator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: python
+
+    from qiskit.opflow import StateFn, PauliExpectation, CircuitSampler
+
+    # Define the state to sample
+    measurable_expression = StateFn(opflow_op, is_measurement=True).compose(opflow_state)
+
+    # Convert to expectation value calculation object
+    expectation = PauliExpectation().convert(measurable_expression)
+
+    # Note that there are other expectation value methods: MatrixExpectation(), AerPauliExpectation(), 
+    # but they are used just like PauliExpectation()
+
+Next, the actual calculation is done by the `CircuitSampler` class, which receives a backend or `QuantumInstance` and the expectation object. Here are some examples of how it can be used:
+
+1. Run locally by using an AerSimulator
+*****************************************
+
+.. code-block:: python
+
+   from qiskit.providers.aer import AerSimulator
+
+    # define backend -> local simulator
+    simulator = AerSimulator() 
+
+    # inject backend into circuit sampler
+    sampler = CircuitSampler(simulator).convert(expectation)
+
+    # evaluate
+    expectation_value_2 = sampler.eval().real
+
+    print("sampled: ", expectation_value_2)
+
+2. Run on a remote simulator or real backend
+*********************************************
+
+Here we use the `ibmq_qasm_simulator`, but the workflow is the same when using a real device.
+
+.. code-block:: python
+
+    from qiskit import IBMQ
+
+    IBMQ.load_account()
+    provider = IBMQ.get_provider(hub='ibm-q-internal') # here the hub should not be internal
+    backend = provider.get_backend("ibmq_qasm_simulator")
+
+    # inject backend into circuit sampler
+    sampler = CircuitSampler(backend).convert(expectation) 
+
+    # evaluate
+    expectation_value_4 = sampler.eval().real
+
+    print("sampled: ", expectation_value_4)
+
+Legacy methods (not using opflow)
+---------------------------------
+
+Construct the circuit entirely by yourself and call `backend.run()`.
+
+New method: Use primitives
+-----------------------------
+
+Opflow provided its own classes to represent both operators and quantum states:
+
+.. code-block:: python
+
+    from qiskit.opflow import CircuitStateFn, PauliSumOp
+
+    opflow_op = PauliSumOp(op)
+    opflow_state = CircuitStateFn(state) # convert to a state
```

Comment:
@ElePT - is the "New method - Use primitives" text correct?  It's the same as "Legacy methods (using opflow).

Reply:
you're right, this must have got carried over from the previous section in some copy-paste, it's not used anywhere. Let's remove it.